.github/workflows/validate-generated-files.yml

```diff
@@ -0,0 +1,23 @@
+name: validate-generated-files
```

Comment:
We're missing one more action - https://github.com/ConduitIO/conduit/blob/main/.github/workflows/project-management.yml. It can be copied over as is.

Reply:
added.